src/main/java/dev/cigarette/agent/ZombiesAgent.java

```diff
@@ -136,9 +137,10 @@ private double estimateZombieSpeed(ZombiesAgent.ZombieTarget zombie) {
         return switch (zombie.type) {
             case ZOMBIE, SKELETON, CREEPER, WITCH -> 5.0; // ~0.25 B/t * 20 t/s
             case BLAZE -> 8.0;
-            case WOLF -> 7.0;
+            case WOLF, IRON_GOLEM -> 7.0;
```

Comment:
do we plan to merge this before the experimental branch? if so this should be fine